README.md

```diff
@@ -1,280 +1,70 @@
-
-```
-      # When web.execution.time.min.raw value is Long.MAX_VALUE return 0.
-      - name: web.execution.time.min
-        source: func:IfThenElse(web.execution.time.min.raw, =, 0x7fffffffffffffffL, 0L, metric:web.execution.time.min.raw)
-        type: long_gauge
-        label: min servlet processing time
-        description: Minimum execution time of all servlets in this context
-        unit: ms
-```
-
-- IfThenElseString - similar to IfThenElse function except it produces literal value as result and can be used both on metrics and on tags.
-
-```
-      - name: tomcat.web.app
-        value: func:IfThenElseString(webapp_name,, /, tag:webapp_name)
-```
+The agent supports a number of [built-in functions](/docs/built-in-functions.md) to process the collected metrics before 
+sending it to output. You can also plugin-in custom functions. 
 
-- LongDivide - like DoubleDivide except it produces Long value as result.
-- LongMultiply - like DoubleMultiply except it produces Long value as result.
-- LongMultiplyWithConstant - like DoubleMultiplyWithConstant except it produces Long value as result.
+## Getting Started
 
-```
-      - name: cache.size
-        source: func:LongMultiplyWithConstant(cache.size.kb,1024)
-        type: long_gauge
-        label: size
-        description: current cache size
-        unit: bytes
-```
-
-- LongSubtract - produces result of long type calculated by subtracting second param from first param
-
-```
-      - name: batches.received.scheduling.delay
-        source: func:LongSubtract(receivedStart,receivedSubmit)
-        type: long_gauge
-        label: last received batch scheduling time
-        unit: ms
-```
+### Build
+To build Sematext App Agent you need: 
 
-- SplitAndExtract - Splits the value of specified metric name (first param) using the token (second param) and returns the value at specified index in the array (index being third param). Can be used both on metrics and tags.
+1) Linux based Operating System 
+2) Java 1.6+
+3) Maven 
+4) Thrift compiler v0.9.3
+5) fpm package manager
 
-```
-    tag:
-      - name: tomcat.connector.port
-        value: func:SplitAndExtract(host_port,-,1)
-```
-
-- StringEquals - Compares string values. First argument is field name. Second is string which should be used to compare. Optional third argument is ignore case. By default ignore case is false. Returns 1/0 for true/false, based on the result of equals.
-```
-      - name: upstream.state.up
-        source: func:StringEquals(upstream.server.state,up)
-        type: gauge
-        label: state up
-        description: Server is up
-```
+After cloning the repo, executing `build.sh` will build the packages for multiple Linux distributions.
 
-- TimeSince - Returns time in ms passed since event was accured. Simple calculation of now() - time_contained_in_metric_param
+### Setup
+The packages can be installed using OS specific package manager like dpkg, yum, etc. Once installed a new App can be 
```

Comment:
I think we need some quick explanation of what is an "App".

Reply:
@megastef will have a What's an App video ready soon.
@goranbajic worked on a Monitoring Guide (https://sematext.com/docs/guide/monitoring-guide/) where I now hoped to see the concept of an App explained, but it looks like we didn't include that (yet?).